proposals/stack-switching/Explainer.md

```diff
@@ -884,7 +898,7 @@ H^ea ::=
   label_n{instr*} H^ea end
   frame_n{F} H^ea end
   catch{...} H^ea end
-  prompt{hdl*} H^ea end   (iff ea notin tagaddr(hdl*))
```

Comment:
Not sure I understand this change, you need to first compute the set of free tag addresses.